lib/test.js

```diff
@@ -92,22 +102,50 @@ function run(state, callback) {
     process.env.DISPLAY = program.display ? program.display : ':0';
   }*/
 
+  if(command.sauceConfig) {
+    remoteSelenium = true;
+    if(!(process.env.SAUCE_USERNAME && process.env.SAUCE_ACCESS_KEY)) {
+      return callback(new Error(
+        'Missing environment variables: SAUCE_USERNAME, SAUCE_ACCESS_KEY'));
+    }
+    bedrock.config.protractor.meta = {
+      build: command.sauceBuild || null,
+      name: command.sauceName || null,
+      tags: []
+    };
+    if(command.sauceTag) {
+      command.sauceTag.split(',').forEach(function(t) {
+        bedrock.config.protractor.meta.tags.push(t);
+      });
+    }
+    bedrock.config.protractor.config.directConnect = false;
+    bedrock.config.protractor.config.sauceUser = process.env.SAUCE_USERNAME;
+    bedrock.config.protractor.config.sauceKey = process.env.SAUCE_ACCESS_KEY;
+    // increase maxTimeout due to additional latency when using SauceLabs
+    bedrock.config.protractor.config.params.config.maxTimeout = 30000;
+    command.sauceConfig.split(',').forEach(function(c) {
+      require(path.join(process.cwd(), c))(bedrock);
+    });
+  }
+
   // generate the protractor config
   var configFile = generateProtractorConfig();
 
   // run protractor
   var browser = bedrock.config.protractor.config.capabilities.browserName;
   console.log('Running tests via Protractor...');
   console.log('Server: ' + bedrock.config.server.baseUri);
-  console.log('Browser: ' + browser);
-  if(bedrock.config.protractor.config.capabilities.chromeOptions &&
-    bedrock.config.protractor.config.capabilities.chromeOptions.binary) {
-    console.log('Chrome binary: ' +
+  if(!remoteSelenium) {
+    console.log('Browser: ' + browser);
+    if(bedrock.config.protractor.config.capabilities.chromeOptions &&
+      bedrock.config.protractor.config.capabilities.chromeOptions.binary) {
+      console.log('Chrome binary: ' +
       bedrock.config.protractor.config.capabilities.chromeOptions.binary);
-  }
-  if(bedrock.config.protractor.config.firefoxPath) {
-    console.log('Firefox binary: ' +
+    }
+    if(bedrock.config.protractor.config.firefoxPath) {
+      console.log('Firefox binary: ' +
       bedrock.config.protractor.config.firefoxPath);
+    }
   }
```

Comment:
As soon as the test starts up, it indicates that it's using a remote selenium server etc.  Not sure we need anything more than that.  What did you have in mind?